requirements.txt

```diff
@@ -1,6 +1,7 @@
 Sphinx==1.3.4
 dj-database-url==0.3.0
 flake8==2.4.0
+inflection==0.3.1
```

Comment:
No problem, this is definitely not intuitive. I'll add some comments to requirements.txt making this clear.